hyp3_gamma/insar/unwrapping_geocoding.py

```diff
@@ -109,7 +111,7 @@ def geocode(inname, outname, inwidth, lt, outwidth, outlines, type_):
     execute(f"geocode {lt} {inname} {inwidth} {outname} {outwidth} {outlines} - {type_}", uselogging=True)
 
 
-def data2geotiff(inname, outname, dempar, type_):
+def data2geotiff(inname, outname, dempar, type_, shift_origin=False):
```

Comment:
`shift_origin` should be removed if it isn't being used
```suggestion
def data2geotiff(inname, outname, dempar, type_):
```